skel/lib/systemd/system/dcache.service

```diff
@@ -0,0 +1,14 @@
+#systemd service for managing all dCache domains on the system. This service is
+#actually a systemd target, but as targets cannot be reloaded a service is used.
+
+[Unit]
+Description=dCache
+
+[Service]
+Type=oneshot
+ExecStart=/bin/true
```

Comment:
Shouldn't it be the dcache start command  ? same comment for below

Reply:
No, the "global" unit, will pull in the instance units (i.e. the dcache@domain.service units)...Actually in my opinion, dcache start/stop should eventually fully go away (and at least on systemd system already start to give warnings that it will be phased out).

Reply:
yes you're right and my tests to validate the pull requests succeeded: sudo systemctl start  dcache.service ; sudo systemctl status dcache.service; sudo systemctl stop dcache.service

Reply:
I already tried to start that in the other issue,... I think dCache developers should decide on how they want to implement systemd support,... i.e. properly or just making it work somehow.

The proper way likely implies that some things dCache does now itself (e.g. the settings mentioned in the other ticket, or dcache start/stop) is mostly moved out of dCache itself.
For as long as we need to support SL6 I'd suggest to rather make wrappers for that, than to hold development towards the direction where things will sooner or later anyway go to. :)